src/components/CountdownPane/CountdownPane.js

```diff
@@ -59,7 +59,7 @@ export const CountdownPane = ({etd, err, dob}) => {
             value={percentage} 
             strokeWidth={2}
             styles={buildStyles({strokeLinecap: 'butt', 
-            pathColor: `#4d7880`, 
+            pathColor: `hsla(174, 86%, 57%, 0.3)`, 
```

Comment:
I still wonder if the path should be going the other way?

Reply:
oh right! I can revisit that when we are refactoring.